core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -23,6 +23,7 @@
  */
 package hudson.slaves;
 
+import com.google.common.io.NullOutputStream;
```

Comment:
As in https://github.com/jenkinsci/workflow-cps-plugin/pull/231 you should not use this.

Reply:
https://issues.jenkins-ci.org/browse/JENKINS-51889